### PR TITLE
Settings screen

### PR DESCRIPTION
**Pull request by AlexanderBrockmann**
Implemented a Settings button/screen (so far without any features) and fixed a bug where you couldn't start a new game after you've finished one game.

### CODE REVIEW

**Comment by AlexanderBrockmann** [comment]
fixes #67

File: core/src/de/adesso/brainysnake/BrainySnake.java [hunk 1]
```diff
@@ -271,10 +280,52 @@ public void clicked(InputEvent event, float x, float y) {
...
+            public void clicked(InputEvent event, float x, float y) {
+                hideAllActors(mainStage);
+                menuShowing = false;
+                settingsShowing = true;
+            }
+        });
+        mainStage.addActor(settingsButton);
+
+        gameMaster.setGameOver(false);
+        GlobalGameState.resetMoves();
```
Comment (FrederikSchlemmer) [inline core/src/de/adesso/brainysnake/BrainySnake.java:296]: **Enhancement:** Create an own reset method and call this method at the "Back to Menu" button event.

File: core/src/de/adesso/brainysnake/BrainySnake.java [hunk 2]
```diff
@@ -397,6 +448,8 @@ public void drawGameOverScreen() {
         pixmap.setColor(Color.WHITE);
         pixmap.fill();
 
+        //gameMaster.setGameOver(false);
```
Comment (FrederikSchlemmer) [inline core/src/de/adesso/brainysnake/BrainySnake.java:451]: Remove code, which is commented out.

File: core/src/de/adesso/brainysnake/BrainySnake.java [hunk 3]
```diff
@@ -133,6 +137,11 @@ public void render() {
             return;
         }
 
+        if(settingsShowing){
+            drawSettingsScreen();
```
Comment (FrederikSchlemmer) [inline core/src/de/adesso/brainysnake/BrainySnake.java:141]: Settings screen seems to useless in the current version. Add with included functionality in version 1.1.

**Pull request closed without merging**